source/server/admin/stats_params.h

```diff
@@ -29,6 +49,13 @@ struct StatsParams {
    */
   Http::Code parse(absl::string_view url, Buffer::Instance& response);
 
+  /**
+   * @return a string representation for a type.
+   */
+  static absl::string_view typeToString(StatsType type);
+
+  StatsType type_{StatsType::All};
+  StatsType start_type_{StatsType::TextReadouts};
```

Comment:
Where is this used?

Reply:
good catch! I think it is used in the follow-up PR but will add it there. Removed.